### ISSUE TITLE
importing element_line

### ISSUE THREAD

**Issue by gdmcbain** [kinnala/scikit-fem#416]
Puzzlingly, after running ex29 (primitive Orr&ndash;Sommerfeld equation) many many times myself, I had a user report:

> Specifically, I have Python 3.8.2. Running your example using that gives:

```
$ python3 ex29_27e3.py
Traceback (most recent call last):
  File "ex29_27e3.py", line 2, in <module>
   import skfem.element.element_line as element_line
ImportError: cannot import name 'element_line' from 'skfem.element.element' (/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/skfem/element/element.py
```

Surprised and not really knowing what was going on at that end, I put together a Binder (currently https://mybinder.org/v2/gh/gdmcbain/skfem-binders/prorso, but that mightn't last long) and got the same error!

```
Traceback (most recent call last):
  File "ex29.py", line 56, in <module>
    import skfem.element.element_line as element_line
ImportError: cannot import name 'element_line' from 'skfem.element.element' (/srv/conda/envs/notebook/lib/python3.7/site-packages/skfem/element/element.py)
```

Then I created a fresh conda env (Python 3.8.3) and with `pip install scikit-fem` (which I presume the user had done) again got the same error:
```
Traceback (most recent call last):
  File "/home/gmcbain/src/scikit-fem/docs/examples/ex29.py", line 56, in <module>
    import skfem.element.element_line as element_line
ImportError: cannot import name 'element_line' from 'skfem.element.element' (/home/gmcbain/.local/share/miniconda3/envs/prorso-import/lib/python3.8/site-packages/skfem/element/element.py)
```

If I then do
```
$ pip install ~/src/scikit-fem[full] 
```
it works fine!

What have I done wrong here?  Is it in the line
```python
    import skfem.element.element_line as element_line
```
or in the hierarchy of the packages and `__init__.py` files?

It's annoying that tests.test_examples.TestEx29 passes; I guess that goes back to how to test installed code #21.

**Comment by gdmcbain** [comment]
Is it just a lag in pushing a version to PyPI?  I also find that in a pip-installed scikit-fem,
```shell
python -c 'from skfem import ElementLineMini'
```
fails with
```
from skfem import Element

**Comment by gdmcbain** [comment]
Yes, 21eca038a &lsquo;export ElementLineMini&rsquo; appears not yet to have made it to PyPI.

**Comment by kinnala** [comment]
I can release 1.2.0 later today.

**Comment by gdmcbain** [comment]
Thanks, that'll do.  Meanwhile I've gotten the user going again with a simplified version of the example.

**Comment by gdmcbain** [comment]
Fixed by [release 1.2.0](https://github.com/kinnala/scikit-fem/releases/tag/1.2.0). Thanks!
